environment.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
 - pytest-cov=2.10.1
 - pytest=6.1.2
 - python=3.8.6
-- pyvirtualdisplay=0.2.5
+- pyvirtualdisplay=1.3.2
```

Comment:
Please don't edit the enviroment.yaml directly.
Instead have a look at [this](https://github.com/mozilla/OpenWPM/blob/master/CONTRIBUTING.md#managing-requirements)

Reply:
Sorry--done.